### PR TITLE
Add aggregator types for nervous system parameters

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

The SNS aggregator has exposed `nervous_system_parameters` since https://github.com/dfinity/nns-dapp/pull/4014 but it wasn't used in the NNS dapp frontend yet.
We want to start using it instead of loading the params separately.

This PR just adds the types to the CachedSnsDto.
This PR does not implement any kind of conversion.
The plan is to replace the `snsParametersStore` with a derived store based on this data, rather than exposing it in the `SnsSummary`.

# Changes

1. Manually added types to `frontend/src/lib/types/sns-aggregator.ts` based on the [Rust type](https://github.com/dfinity/nns-dapp/blob/712fd337ae5cf6cdcfae39badc5a389751f67720/rs/sns_aggregator/src/types/ic_sns_governance.rs#L85-L106).

# Tests

1. Updated `frontend/src/tests/mocks/sns-aggregator.mock.json` by copying `frontend/src/tests/workflows/Launchpad/sns-agg-page-0.json` to make sure that the mock defined [here](https://github.com/dfinity/nns-dapp/blob/712fd337ae5cf6cdcfae39badc5a389751f67720/frontend/src/tests/mocks/sns-aggregator.mock.ts#L13) has the required `nervous_system_parameters` fields.
2. Also copied `nervous_system_parameters` from the above files into `frontend/src/tests/lib/utils/sns-aggregator-converters.utils.spec.ts` to satisfy the type of the input data to the conversion test.

# Todos

- [ ] Add entry to changelog (if necessary).
not necessary

### CODE REVIEW

File: frontend/src/lib/types/sns-aggregator.ts [hunk 1]
```diff
@@ -42,6 +42,48 @@ export type CachedNervousFunctionDto = {
...
+};
+
+export type CachedVotingRewardsParametersDto = {
+  final_reward_rate_basis_points?: number;
+  initial_reward_rate_basis_points?: number;
+  reward_rate_transition_duration_seconds?: number;
+  round_duration_seconds?: number;
+};
+
+export type CachedNervousSystemParametersDto = {
```
Comment (lmuntaner) [inline frontend/src/lib/types/sns-aggregator.ts:64]: Why are all optional?

**Review by lmuntaner — COMMENTED** [review]
Just one question.

File: frontend/src/lib/types/sns-aggregator.ts (hunk 1 above)
Comment (dskloetd) [reply frontend/src/lib/types/sns-aggregator.ts:64]: In the [Rust type](https://github.com/dfinity/nns-dapp/blob/712fd337ae5cf6cdcfae39badc5a389751f67720/rs/sns_aggregator/src/types/ic_sns_governance.rs#L85-L106), the are all optional. So they should be optional here as well, no?

Comment (lmuntaner) [reply frontend/src/lib/types/sns-aggregator.ts:64]: That's up to us, but they are optional for backwards compatiblity reason. If this bothers your further types, you might want to rething and not have everything optional though. But you'll need to deep dive into each field.

I guess it's the safest and rational choice though...

**Review by lmuntaner — APPROVED** [review]

File: frontend/src/lib/types/sns-aggregator.ts (hunk 1 above)
Comment (dskloetd) [reply frontend/src/lib/types/sns-aggregator.ts:64]: OK, thanks. I'll keep it in mind and potentially change it later if.

**Review by mstrasinskis — APPROVED** [review]

**Pull request merged**